AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/AnyModelIntegrationTests.swift

```diff
@@ -42,10 +42,9 @@ class AnyModelIntegrationTests: XCTestCase {
     }
 
     func testCreateAsAnyModel() throws {
-
-        let originalPost = Post(title: "Post title",
-                                content: "Original post content as of \(Date())",
-                                createdAt: Date())
+        let originalPost = AmplifyTestCommon.Post(title: "Post title",
```

Comment:
is there a conflict with `Post` so you needed to use the fully qualified name? I wonder if we could remove the conflict instead.

Reply:
i did some clean up here and removed the non-Model based Blog/Post/Comment types which have tests that were also removed (they were disabled, other Model based integ tests cover the same scenario) so that we can remove the need for the fully qualified name here and rest of the files